### PR TITLE
Run superset tests on travis

### PR DESCRIPTION
**Pull request by smacker**
fix: #86

### CODE REVIEW

File: superset/requirements-dev.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ pip-tools==3.6.1
 psycopg2-binary==2.7.5
 pycodestyle==2.4.0
 -e git+https://github.com/src-d/PyHive.git@v0.7.0-srcd1#egg=PyHive
+requests_kerberos>=0.12.0
```
Comment (carlosms) [inline superset/requirements-dev.txt]: why this new requirement?

Comment (carlosms) [reply superset/requirements-dev.txt]: ok, I've seen the commit message now

> this package needed to pass `presto` tests using pyhive

**Comment by carlosms** [comment]
I did a quick test:
- Merged your branch to `origin/tmp-smacker-travis`
- Forced a mocked release (fb038a90, 3b805f43)

In the resulting travis build I see 2 problems:
https://travis-ci.com/src-d/sourced-ui/builds/114137033
- The `release` stage was executed before `tests`. If I recall correctly, the jobs without a explicit `stage` name fall into `test`, not `tests`
- The build failed trying to run `pip install`. Maybe the `release` job needs to also have `python: 3.6` defined?

**Comment by smacker** [comment]
Thank you @carlosms !
Fixed. I also rebased tmp branch and now it's green: https://travis-ci.com/src-d/sourced-ui/builds/114256225

**Pull request merged**